docs/tasks/README.md

```diff
@@ -3,6 +3,23 @@
 This section contains pages that show how to perform individual tasks.
```

Comment:
The following section includes the tasks available for the following user personas:

* Batch administrators
* Batch users